From b9d7bd5c8042ff054e05754fbbc9af752790556e Mon Sep 17 00:00:00 2001
From: Zidan Wang <b50113@freescale.com>
Date: Fri, 19 Dec 2014 11:43:47 +0800
Subject: [PATCH 1113/1543] MLK-10038-2: ARM: dts: Add "revision-a10" to set
 command set to A10

commit bbca8e713c2d5832c33000f3d833cabfbb941b20 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add property "revision-a10" to device tree to set the default command
set to A10.

Signed-off-by: Zidan Wang <b50113@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    1 +
 arch/arm/boot/dts/imx6sx-sabreauto.dts   |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 3f838be..7186f6e 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -447,6 +447,7 @@
 		vd-supply = <&reg_si4763_vd>;
 		vio1-supply = <&reg_si4763_vio1>;
 		vio2-supply = <&reg_si4763_vio2>;
+		revision-a10; /* set to default A10 compatible command set */
 
 		si476x_codec: si476x-codec {
 			compatible = "si476x-codec";
diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 1c64315..be1cb26 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -578,6 +578,7 @@
 		vd-supply = <&si4763_vd>;
 		vio1-supply = <&si4763_vio1>;
 		vio2-supply = <&si4763_vio2>;
+		revision-a10; /* set to default A10 compatible command set */
 
 		si476x_codec: si476x-codec {
 			compatible = "si476x-codec";
-- 
1.7.5.4

